frontend/public/components/utils/units.js

```diff
@@ -101,20 +96,27 @@ const convertValueWithUnitsToBaseValue = (value, unitArray, divisor) => {
   return { value, unit };
 };
 
-const round = units.round = (value) => {
-  if (!isFinite(value)) {
-    return 0;
+const getDefaultFractionDigits = value => {
+  if (value < 1) {
+    return 3;
   }
-  let digits;
-  if (value >= 100) {
-    digits = 1;
-  } else if (value >= 1) {
-    digits = 2;
-  } else {
-    digits = 3;
+  if (value < 100) {
+    return 2;
   }
-  const multiplier = Math.pow(10, digits);
-  return Math.round(value * multiplier) / multiplier;
+  return 1;
+};
+
+const round = units.round = (value, options) => {
+  const fractionDigits = getDefaultFractionDigits(value);
+  const {locales, ...rest} = _.defaults(options, {
+    locales: 'en-US',
```

Comment:
I don't think we want to default locale. If we leave it out, the system locale is used.

> The locales argument must be either a string holding a BCP 47 language tag, or an array of such language tags. If the locales argument is not provided or is undefined, the runtime's default locale is used.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation

Reply:
Fixed